docs/reference/modules/indices/query_cache.asciidoc

```diff
@@ -1,27 +1,35 @@
 [[query-cache]]
 === Node Query Cache
 
-The query cache is responsible for caching the results of queries.
-There is one queries cache per node that is shared by all shards.
-The cache implements an LRU eviction policy: when a cache becomes full, the
-least recently used data is evicted to make way for new data.
-It is not possible to look at the contents being cached.
+The query cache is responsible for caching the results of queries. There is one 
+queries cache per node that is shared by all shards. The cache implements an LRU 
+eviction policy: when a cache becomes full, the least recently used data is 
+evicted to make way for new data. It is not possible to look at the contents 
+being cached.
 
-The query cache only caches queries which are being used in a filter context.
+The query cache only caches queries which are being used in a filter context. 
+However, term queries are not eligible for caching, not even in the filter 
+context.
+
+It caches up to 10000 queries in up to 10% of the total heap space by default. 
+To determine if a query is eligible for caching, a history of queries is 
+maintained to track occurrences.
+
```

Comment:
I'd probably switch this to the active voice:

To determine if a query is eligible for caching, Elasticsearch maintains a query history to track occurrences.